docs/source/tutorial/peft_integrations.md

```diff
@@ -141,10 +141,12 @@ output = model.generate(**inputs)
 print(tokenizer.decode(output_disabled[0], skip_special_tokens=True))
 ```
 
-To disable the adapter, call the [`~PeftModel.disable_adapter`] method.
+To disable the adapter, call the `disable_adapters` method.
 
 ```py
-model.disable_adapter()
+model.disable_adapters()
 ```
 
+The `enable_adapters` can be used to enable the adapters again.
```

Comment:
```suggestion
The [`~diffusers.loaders.PeftAdapterMixin.enable_adapters`] can be used to enable the adapters again.
```

Reply:
Ah sorry, there are so many `enable_adapters` everywhere haha! Yeah, it should be [`~transformers.integrations.enable_adapters`] then, and the same for `disable_adapters`.

Reply:
I tried:

```
[`~transformers.integrations.PeftAdapterMixin.enable_adapters`]
```

but the link is not added in the preview doc. Any idea what the issue might be?

Reply:
Indeed there is no hit when I search for the methods in the transformers documentation. So what should I link to, a perma link to the source code?

Reply:
Yeah that sounds good to me!